src/backend/test/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ target_link_libraries(unit_tests_backend
     mavsdk_offboard
     mavsdk_telemetry
     gRPC::grpc++
+    protobuf
```

Comment:
I'm not sure if that's the right way to do it (mainly because the CI fails). Normally it comes in the `gRPC::grpc++` target, I would say. I wonder if `FindgRPC.cmake` should not take care of this (but I'm not sure how)

Reply:
The original FindGRPC had something for protobuf but I did not take the time to understand what it did:
https://raw.githubusercontent.com/wastl/cmarmotta/master/cmake/FindgRPC.cmake